src/components/elements/query-image.tsx

```diff
@@ -58,11 +44,9 @@ const QueryImage = ({
     if (data) {
         return (
             <ImageWrapper
-                loading={loading}
```

Comment:
The `loading` property is still used in multiple places. Some functionality will break if you remove this property. 

A few explanation for this property: It is an extension of the base type of `gatsby-plugin-image` https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#shared-props

You can check this by searching for `loading="eager"`